tests/components/sleepiq/conftest.py

```diff
@@ -0,0 +1,59 @@
+"""Common fixtures for sleepiq tests."""
+
+import pytest
+
+from homeassistant.components.sleepiq.const import DOMAIN
+from homeassistant.const import CONF_PASSWORD, CONF_SCAN_INTERVAL, CONF_USERNAME
+
+from tests.common import MockConfigEntry, load_fixture
+
+BASE_URL = "https://prod-api.sleepiq.sleepnumber.com/rest/"
+
+
+@pytest.fixture
+def requests_mock_fixture(request, requests_mock):
```

Comment:
We should mock the pypi package and not requests since at some point the package may no longer use requests